### ISSUE TITLE
Keyboard navigation doesn't work on a input with initialValue

### ISSUE THREAD

**Issue by maheshj01** [maheshj01/searchfield#179]
When the Searchfield has a entered value, 

1. Focus the Searchfield
2. use arrow keys to navigate suggestions

expected behavior: Suggestion can be navigated with arrow keys

actual behavior: Nothing happens

**Comment by maheshj01** [comment]
This issue seems, bit Funky unable to reproduce it most of the times.

**Comment by sabari7320** [comment]
@maheshj01  sir when  i close the keyboard all the suggestion showing from scratch not showing hat i have entered char in the textfield
https://drive.google.com/file/d/132iZYWgLzDNMllLGyOlkA15SayzkrcZe/view check this

**Comment by maheshj01** [comment]
@sabari7320  Please file a separate issue with all the details

**Comment by maheshj01** [comment]
Regression identified https://github.com/maheshj01/searchfield/commit/0fddae486c8538307b9719e9230739733d5db378#diff-0d4d907e62a927a9256ddab70f46e08912d122e3b1634116aa27498cc43a2db8R447

**Comment by maheshj01** [comment]
Fixed in latest release [v1.1.9](https://pub.dev/packages/searchfield/changelog)
